### PR TITLE
fixes #1103 - File Entity Page: Export of Associated Particpant Table is odd

### LINKED ISSUE #1103: File Entity Page: Export of Associated Particpant Table is odd

### PR DESCRIPTION
**Pull request by jberube**

### CODE REVIEW

**Comment by jberube** [comment]
The thing is that it's not an array, it's a React Component. Study Name is a link when displayed.

`data` contains that Component. What I did is decouple the data from it's representation so the TSV export deals with raw data only.

File: src/components/EntityPage/File/index.js [hunk 1]
```diff
@@ -331,6 +333,24 @@ const FileEntity = ({ api, fileId, isPageLoading, hasFilePermission }) => (
...
+                              TRACKING_EVENTS.actions.click +
+                              `: Associated Participants/Biospecimens: Study Name`,
+                            label: studyShortName,
+                          });
+                        }}
+                      >
+                        {studyShortName}
+                      </ExternalLink>
+                    ),
+                  }}
```
Comment (hlminh2000) [inline src/components/EntityPage/File/index.js]: @ciaranschutte this actually looks good to me, I like this so we can keep data separate from presentation. The bug was caused by React components being mixed in with the data that gets pulled into the export file :)

**Comment by rosibaj** [comment]
@hlminh2000 @jberube i checked out the preview - this looks great.  Lets get it merged to VR and we can close the ticket

**Comment by jberube** [comment]
I'll test the branch locally after tomorrow. The merge has not been trivial and I can't test locally from home.

**Comment by jberube** [comment]
Everything's in order. Can be merged.

**Comment by ciaranschutte** [comment]
lgtm merging

**Pull request merged**